pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -840,6 +840,8 @@ async def publish_multi_client(self, base_to_mirror_dir: str, pullspec: str, rel
             os.makedirs(client_mirror_dir)
             # extract release clients tools
             extract_release_client_tools(pullspec, f"--to={client_mirror_dir}", go_arch)
+            # extract baremetal installer binary
+            self.publish_baremetal_installer_binary(pullspec, client_mirror_dir, arch)
```

Comment:
Not tested, but seems `arch` should be `goarch`.

Reply:
I don't think so, because it's converted to `goarch` inside `publish_baremetal_installer()`: https://github.com/openshift-eng/art-tools/blob/1002eac91ecd6da518275a47fdf191ee7c6c7e2e/pyartcd/pyartcd/pipelines/promote.py#L790